dbt/tests/generic/test_sequential_values.sql

```diff
@@ -8,7 +8,7 @@
     additional_select_columns=[]
 ) %}
 
-    {%- set previous_column_name = "previous_" ~ dbt_utils.slugify(column_name) -%}
+    {%- set previous_column_name = "previous_" ~ slugify(column_name) -%}
```

Comment:
Our version of slugify is basically identical to the `dbt_utils` version; the only differences are that we convert backslashes to underscores and we don't prepend underscores if the string starts with a number. Neither of those differences should matter in this case since our column names don't contain backslashes or start with numbers.

For reference, here's our version of the macro:

https://github.com/ccao-data/data-architecture/blob/6efd15a7b874cd8aa403bbd02c704398344e1020/dbt/macros/slugify.sql#L1-L13

And[ here's the dbt utils version](https://github.com/dbt-labs/dbt-utils/blob/fc98e4e30cd299ddd67c43902f2a5efa9511edfb/macros/jinja_helpers/slugify.sql#L1-L14):

```sql
{% macro slugify(string) %}


{#- Lower case the string -#}
{% set string = string | lower %}
{#- Replace spaces and dashes with underscores -#}
{% set string = modules.re.sub('[ -]+', '_', string) %}
{#- Only take letters, numbers, and underscores -#}
{% set string = modules.re.sub('[^a-z0-9_]+', '', string) %}
{#- Prepends "_" if string begins with a number -#}
{% set string = modules.re.sub('^[0-9]', '_' + string[0], string) %}


{{ return(string) }}


{% endmacro %}
```